static/sidebar/js/modal/index.js

```diff
@@ -54,14 +54,19 @@
       prodNode.href = productLink;
       prodNode.innerHTML = '<div class="tvp-product-image" '+productImgStyle+'><div/>';
       thumbsFrag.appendChild(prodNode);
+      
+      var trimmedTitle = product.title.length > 50 ? product.title.substring(0, 50 - 3) + "..." :product.title;
+      var price = product.price.toString().replace(/[^0-9.]+/g, '');
+      price = parseFloat(price).toFixed(2);
```

Comment:
first remove special characters and leave just numbres (in case there's double "$" sign)
thne if the ammount has multiple numbers ie. "59.6546" = "59.65"